sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -541,8 +540,7 @@ void gnrc_ndp_rtr_adv_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt, ipv6_hdr_t
     }
     /* set retransmission timer from message */
     if (rtr_adv->retrans_timer.u32 != 0) {
-        if_entry->retrans_timer = timex_set(0, byteorder_ntohl(rtr_adv->retrans_timer));
-        timex_normalize(&if_entry->retrans_timer);
+        if_entry->retrans_timer = byteorder_ntohl(rtr_adv->retrans_timer);
```

Comment:
This is wrong. The [retransmission timer](https://tools.ietf.org/html/rfc4861#page-21) in a RA is send in milliseconds, while it is stored (and used as) a microsecond value in `gnrc_ipv6_netif_t::retrans_timer`. Can you maybe fix that in an a follow-up PR (+ hot-fix to the release branch)